docs/introduction.md

```diff
@@ -0,0 +1,7 @@
+# Introduction
+
+This guide will help users of Paricia to get up to speed with its functionality and start making use of its features.
+
+There are two types of Paricia users, those who are interested in exploring the data available - either registered users or not - and those who manage stations and want to create new entries in the database. The first group can just check the [Reporting](./reports.md) documentation, as it contains all the information they will need. The others should become familiar with all the pages of this user guide.
+
+It should not be necessary to check the Developers documentation, but it might be helpful, occasionally, to have a look at the [Contributing guidelines](./contributing.md) if you spot something that doest not work as expected and you want to report it, and at the [Applications page](./Applications/index.md) where there are mode detailed descriptions of the models that make up Paricia.
```

Comment:
```suggestion
It should not be necessary to check the Developers documentation, but it might be helpful, occasionally, to have a look at the [Contributing guidelines](./contributing.md) if you spot something that does not work as expected and you want to report it, and at the [Applications page](./Applications/index.md) where there are mode detailed descriptions of the models that make up Paricia.
```